### ISSUE TITLE
Must call command palette twice to get a command to work

### ISSUE THREAD

**Issue by ucirello** [Microsoft/vscode#60568]
- VSCode Version: Versione 1.28.0 (1.28.0)
- OS Version: Mojave

Steps to Reproduce:

1. Open a file
2. Call some command in the command palette, it ignores you.
3. Call the same command again, it works.

Does this issue occur when all extensions are disabled?: Yes/No
No

**Comment by egamma** [comment]
Cannot reproduce based on the steps given above. Can you please add an animated GIF that illustrated the problem.

**Comment by egamma** [comment]
The action 'Change case upper' is not provided by VS Code.
The VS Code provided action is 'Transform to upper' does the problem happen with this action as well?

**Comment by ucirello** [comment]
@egamma For the VSCode native commands, it works just fine.

But for every extension command, I find this problem rather frequently. For instance, sometimes the vscode-go format on save fails, and I need to force saving couple of times to succeed.

The same goes for the `change uppercase` which comes from an extension, or the FileUtils commands such delete or duplicate file.

**Comment by ucirello** [comment]
@egamma For testing purposes, I disabled all extensions and enabled just one (`wmaurer.change-case`) and tried again. I managed to reproduce the bug almost consistently. It does not affect only `wmaurer.change-case`, virtually every other extension I have.

**Comment by egamma** [comment]
@ucirello when you have disabled all extensions can you provide one sequence of steps that demonstrates the problem.

**Comment by egamma** [comment]
@ucirello when you have disabled all extensions can you provide one sequence of steps that demonstrates the problem.

**Comment by ucirello** [comment]
1. Install `wmaurer.change-case`
2. Disable all extensions except `wmaurer.change-case`
3. Close all folders (CMD+K F)
4. Install this settings.json
```
{
	"editor.autoClosingBrackets": false,
	"editor.cursorBlinking": "solid",
	"editor.cursorStyle": "underline",
	"editor.detectIndentation": true,
	"editor.fontSize": 14.5,
	"editor.formatOnSave": false,
	"editor.insertSpaces": false,
	"editor.lineHeight": 17,
	"editor.lineNumbers": "on",
	"editor.minimap.enabled": false,
	"editor.minimap.renderCharacters": false,
	"editor.minimap.showSlider": "mouseover",
	"editor.renderControlCharacters": true,
	"editor.renderIndentGuides": true,
	"editor.renderLineHighlight": "all",
	"editor.renderWhitespace": "all",
	"editor.roundedSelection": true,
	"editor.rulers": [ 80, 120 ],
	"editor.scrollBeyondLastLine": false,
	"editor.smoothScrolling": false,
	"editor.tabSize": 8,
	"editor.trimAutoWhitespace": true,
	"explorer.openEditors.visible": 0,
	"extensions.autoUpdate": true,
	"files.autoSave": "onFocusChange",
	"files.hotExit": "onExitAndWindowClose",
	"files.trimTrailingWhitespace": true,
	"files.watcherExclude": {
		"**/.git/objects/**": true,
		"**/node_modules/**": true,
		"**/.history/**": true
	},
	"files.exclude": {
		"**/web-ro/**": true,
		"**/.git": true,
		"**/.svn": true,
		"**/.hg": true,
		"**/.DS_Store": true,
		"**/.git/objects/**": true,
		"**/.history/**": true,
	},
	"git.enabled": false,
	"go.buildOnSave": "package",
	"go.formatTool": "goimports",
	"go.inferGopath": false,
	"go.lintOnSave": "package",
	"go.testTimeout": "30s",
	"go.vetFlags": [],
	"go.vetOnSave": "package",
	"problems.autoReveal": false,
	"window.nativeTabs": true,
	"window.newWindowDimensions": "maximized",
	"window.openFilesInNewWindow": "off",
	"window.title": "${activeEditorMedium}${separator}${rootName}",
	"workbench.colorTheme": "Go - Playground",
	"workbench.editor.enablePreview": false,
	"workbench.editor.enablePreviewFromQuickOpen": false,
	"workbench.editor.showIcons": false,
	"workbench.editor.showTabs": true,
	"workbench.editor.tabCloseButton": "off",
	"workbench.iconTheme": null,
	"workbench.quickOpen.closeOnFocusLost": false,
	"workbench.startupEditor": "none",
	"explorer.confirmDelete": false,
	"explorer.decorations.colors": false,
	"explorer.decorations.badges": false,
	"go.goroot": "/usr/local/go/",
	"javascript.updateImportsOnFileMove.enabled": "always",
	"editor.multiCursorModifier": "alt",
	"go.useLanguageServer": true,
	"[org]": {

	},
	"terminal.integrated.rendererType": "dom",
	"breadcrumbs.enabled": true,
	"editor.codeLens": false,
	"editor.highlightActiveIndentGuide": false,
	"editor.matchBrackets": false,
	"workbench.statusBar.feedback.visible": false,
	"search.location": "sidebar",
	"terminal.integrated.cursorStyle": "underline",
	"window.zoomLevel": 0,
	"terminal.integrated.cursorBlinking": true,
	"workbench.activityBar.visible": false,
	"workbench.tips.enabled": false,
	"extensions.showRecommendationsOnlyOnDemand": true,
	"terminal.integrated.copyOnSelection": true,
	"terminal.integrated.enableBell": true,
	"git.autorefresh": false,
}
```
5. Close all files and close VSCode.
6. Open VSCode.
7. Create a new file (cmd+N)
8. Manually type:
```
nnn
nnn
nnn
```
9. Select the three lines using shift+up arrow
10. Open command palette (cmd+p)
11. `> Change Case upper`
It fails. Try 9, 10, 11 again. It works.

**Comment by ucirello** [comment]
@egamma It seems that `git.enabled` is the culprit. When I remove it, it works.

**Comment by ucirello** [comment]
It seems that `git.enabled` is the problematic entry. When I turn it to true, many right-bottom popups that were not longer showing are now back.

**Comment by joaomoreno** [comment]
> many right-bottom popups that were not longer showing are now back

What does this mean?

**Comment by ucirello** [comment]
These popups that show in the bottom-right corner.

![image](https://user-images.githubusercontent.com/138827/47582790-ae07e280-d955-11e8-9d52-7bcb97051ee2.png)

None was showing before - once I reenable `git.enabled` they resume showing.

**Comment by ucirello** [comment]
Not only gits - but others too.

**Comment by joaomoreno** [comment]
That is from another extension. Are you absolutely sure all extensions are disabled?

**Comment by ucirello** [comment]
@joaomoreno when I restored the extensions, these toaster messages that were not showing before now do. 

Again, the problem does not replicate with extensions off. You must have at least one extension enabled (and I chose `wmaurer.change-case` for the repro steps) and the native git extension disabled.

**Comment by joaomoreno** [comment]
`gitblame.commitUrl` is not a command VS Code ships.

Maybe try to uninstall all extensions instead of just disabling them. cc @sandy081

**Comment by sandy081** [comment]
Can you also please check if the extension is disabled or not by opening the extensions viewlet? Also open `Show Running Extensions` command and see what extensions are running.

**Comment by ucirello** [comment]
At this point in time I am under the impression that no one tried the step to reproduce the problem. And I think I am not being effective in convey the result of the asked steps. Most importantly, I managed to workaround the problem just by enabling `git.enable:true`. 

All this happens also with fresh VSCode installation. 

I no longer feel I am able to contribute to the investigation, at same time, I am no longer willing to repeat the debug steps (https://github.com/Microsoft/vscode/issues/60568#issuecomment-431751675 and then https://github.com/Microsoft/vscode/issues/60568#issuecomment-434631123 and then https://github.com/Microsoft/vscode/issues/60568#issuecomment-434634898 )

Also I explained that the problem does not replicate with all extensions turned off (https://github.com/Microsoft/vscode/issues/60568#issuecomment-434367466).

As reference for future reader: It seems that there is some weird interaction between VSCode native git module and extensions. When you turn Git support off ( `git.enable:false` in the global settings.json) some instances fail to execute.
